index.js

```diff
@@ -185,6 +185,7 @@ export const Classes = {
 		assignmentActivity: 'assignment-activity',
 		complete: 'complete',
 		completion: 'completion',
+		completionTracked: 'completion-tracked',
```

Comment:
@hekard2l Does `completion-tracked` make sense as being part of the `activities` namespace? `completion` is used only in the activities domain, but seems like `completion-tracked` is part of the `completionTracking` project.

Reply:
Hmm... `completion-tracked` is a class for `Organizations`. I'd personally put it on `line 281`

Reply:
Done! Wasn't really sure where it fit logically-speaking, so thank you! 💯